rs/embedders/benches/compilation.rs

```diff
@@ -214,11 +223,33 @@ fn wasm_validation_instrumentation(c: &mut Criterion) {
     group.finish();
 }
 
+fn execution(c: &mut Criterion) {
+    set_production_rayon_threads();
+
+    let binaries = generate_binaries();
+    // let mut group = c.benchmark_group("execution");
+    // let config = EmbeddersConfig::default();
```

Comment:
Should these be removed? Seems that they are leftovers?

Reply:
Oops, yeah that's left over.